aks-node-controller/parser/helper.go

```diff
@@ -139,26 +145,33 @@ func getKubenetTemplate() string {
 	return base64.StdEncoding.EncodeToString(kubenetTemplateContent)
 }
 
-func getContainerdConfig(aksnodeconfig *aksnodeconfigv1.Configuration) string {
+// getContainerdConfig returns the base64 encoded containerd config depending on whether the node is with GPU or not.
+func getContainerdConfig(aksnodeconfig *aksnodeconfigv1.Configuration, noGPU bool) string {
```

Comment:
a minor side note.

I wonde wherher it's better to perform all base64 conversions in the parser.go.
Just where we assign them to ENV variable.

To have see clearly which of the variables are encoded.

Alternativly, it's worth to add base64 suffix to such methods

Reply:
Another minor thing. I think it will be cleaner to have 2 methods instead of `noGPU bool`

Reply:
Thanks for the suggestions. Updated.